src/mumble/CustomElements.cpp

```diff
@@ -69,6 +69,57 @@ void LogTextBrowser::scrollLogToBottom() {
 	verticalScrollBar()->setValue(verticalScrollBar()->maximum());
 }
 
+void LogTextBrowser::mouseMoveEvent(QMouseEvent *e) {
+	do {
+		QTextCursor cursor = cursorForPosition(e->pos());
+		QTextCharFormat fmt = cursor.charFormat();
+		if (fmt.objectType() == QTextFormat::NoObject) {
+			cursor.movePosition(QTextCursor::NextCharacter);
+			fmt = cursor.charFormat();
```

Comment:
That’s also committed by you.  was able to reproduce the issue and solution. I'll document it.